flink-tests/src/test/java/org/apache/flink/test/checkpointing/UnalignedCheckpointRescaleITCase.java

```diff
@@ -336,38 +335,57 @@ public void create(
         },
         CUSTOM_PARTITIONER {
             final int sinkParallelism = 3;
-            final int numberElements = 1000;
 
             @Override
             public void create(
-                    StreamExecutionEnvironment environment,
+                    StreamExecutionEnvironment env,
                     int minCheckpoints,
                     boolean slotSharing,
-                    int expectedFailuresUntilSourceFinishes,
+                    int expectedRestarts,
                     long sourceSleepMs) {
```

Comment:
The old version was silently ignoring `minCheckpoints`, `expectedRestarts` and `sourceSleepMs` parameters, without throwing any kind of error.